src/tmx/tmx-plugin-config.cmake

```diff
@@ -0,0 +1,111 @@
+endif()
+unset(_targetsDefined)
+unset(_targetsNotDefined)
+unset(_expectedTargets)
+
+
+# Create imported target asn_j2735_r63
+add_library(asn_j2735_r63 SHARED IMPORTED)
+
+set_target_properties(asn_j2735_r63 PROPERTIES
+  INTERFACE_COMPILE_DEFINITIONS "SAEJ2735_SPEC=63"
+  INTERFACE_INCLUDE_DIRECTORIES "/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/Asn_J2735/include/asn_j2735_r63"
+)
+
+# Create imported target tmxapi
+add_library(tmxapi SHARED IMPORTED)
+
+set_target_properties(tmxapi PROPERTIES
+  INTERFACE_COMPILE_DEFINITIONS "BOOST_ASIO_HEADER_ONLY -DBOOST_SPIRIT_THREADSAFE  -DBOOST_ALL_DYN_LINK"
+  INTERFACE_COMPILE_FEATURES "cxx_lambdas;cxx_nullptr;cxx_range_for;cxx_variadic_templates"
+  INTERFACE_INCLUDE_DIRECTORIES "/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/TmxApi;/usr/include"
+  INTERFACE_LINK_LIBRARIES "asn_j2735_r63;/usr/lib/x86_64-linux-gnu/libboost_system.so;/usr/lib/x86_64-linux-gnu/libboost_thread.so;/usr/lib/x86_64-linux-gnu/libboost_atomic.so;/usr/lib/x86_64-linux-gnu/libboost_chrono.so;/usr/lib/x86_64-linux-gnu/libboost_regex.so;/usr/lib/x86_64-linux-gnu/libboost_log.so;/usr/lib/x86_64-linux-gnu/libboost_log_setup.so;/usr/lib/x86_64-linux-gnu/libboost_filesystem.so;/usr/lib/x86_64-linux-gnu/libboost_program_options.so;/usr/lib/x86_64-linux-gnu/libboost_date_time.so;/usr/lib/x86_64-linux-gnu/libpthread.so;/usr/lib/x86_64-linux-gnu/libicui18n.so;/usr/lib/x86_64-linux-gnu/libicuuc.so;/usr/lib/x86_64-linux-gnu/libicudata.so"
+)
+
+# Create imported target tmxutils
+add_library(tmxutils STATIC IMPORTED)
+
+set_target_properties(tmxutils PROPERTIES
+  INTERFACE_INCLUDE_DIRECTORIES "/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/TmxUtils/src;/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/Messages/include;/usr/include/mysql;/usr/include"
+  INTERFACE_LINK_LIBRARIES "tmxapi;/usr/lib/x86_64-linux-gnu/libmysqlclient.so;/usr/lib/x86_64-linux-gnu/libmysqlcppconn.so;/usr/lib/x86_64-linux-gnu/libuuid.so;pthread;m;rt"
+)
+
+# Create imported target tmxctlStatic
+add_library(tmxctlStatic STATIC IMPORTED)
+
+set_target_properties(tmxctlStatic PROPERTIES
+  INTERFACE_INCLUDE_DIRECTORIES "/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/TmxCtl/src/lib"
+  INTERFACE_LINK_LIBRARIES "tmxutils"
+)
+
+# Import target "asn_j2735_r63" for configuration ""
+set_property(TARGET asn_j2735_r63 APPEND PROPERTY IMPORTED_CONFIGURATIONS NOCONFIG)
+set_target_properties(asn_j2735_r63 PROPERTIES
+  IMPORTED_LOCATION_NOCONFIG "/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/lib/libasn_j2735_r63.so"
+  IMPORTED_SONAME_NOCONFIG "libasn_j2735_r63.so"
+  )
+
+# Import target "tmxapi" for configuration ""
+set_property(TARGET tmxapi APPEND PROPERTY IMPORTED_CONFIGURATIONS NOCONFIG)
+set_target_properties(tmxapi PROPERTIES
+  IMPORTED_LOCATION_NOCONFIG "/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/lib/libtmxapi.so"
```

Comment:
same as above

Reply:
I have removed that temporary file. it should be generated again by cmake with correct parameters.